### ISSUE TITLE
Message format not dependend on firmware

### ISSUE THREAD

**Issue by gerherha** [TheRealKillaruna/nano_pk#19]
The message format `<DAQPRJ> ... </DAQPRJ>` is not (just) dependend on the firmware, but changes with different settings made within the Nano-PK. 
The `README.md` is missleading since it states the out of the box support for several firmwares.

The standard case should be to pull the message format in any case. And also after changing major settings if needed.

**Comment by TheRealKillaruna** [comment]
It is not possible to easily pull the message format from the network data, unfortunately. The DAQPRJ block is only written to SD card when logging is enabled, as far as I am aware it is never sent over the network. 
What makes you think that the message format changes depending on your settings?

**Comment by gerherha** [comment]
Yes, by pulling I ment getting the message format with the SD card. Sorry, I should have explained a little more.

Past week I had to change a major setting related to the number of suction points of the pellet storage. The message format was different afterwards, since additional data related to the suction points was sent. 

So I am pretty shure, that the message format is related to the settings and the setup respectively. It is probably also relying on the firmware but not only on it.

I would suggest getting the message format with the SD card should be the standard way. The given message formats within this packet might confuse people if they have the same firmware but it is not working.

**Comment by skrue** [comment]
That would explain why I couldn't get the integration to work with the NANO_V14N2 setting while I was on this exact firmware when initially setting up the integration last year. It wasn't until I defined the `msgformat` manually that I could get it to work. Would make sense to sharpen the documentation a bit in that regard.

**Comment by EpicLPer** [comment]
I actually noticed this today after we had a technician over. The format changed and the integration broke after he seemed to change some settings, despite the same firmware/version still being present.
